src/Data/FunctorF.purs

```diff
@@ -0,0 +1,17 @@
+module Data.FunctorF where
+
+import Data.NaturalTransformation (type (~>))
+
+-- | A type class for higher order functors; types of kind `(* -> *) -> * -> *`.
```

Comment:
I would say "functors in the functor category" since there are many higher-order notions of functor.